CMake/kwiver-setup-tests-python.cmake

```diff
@@ -0,0 +1,38 @@
+#
+# Script to set up environment for python nosetests
+#
+
+
+# results of tests being run will be exported to an Xunit xml file
+if (NOSE_RUNNER)
+
+  set(no_install TRUE)
+  string(TOLOWER "${CMAKE_PROJECT_NAME}" project_name)
+
+  if (WIN32)
+    if(VENV_CREATED)
+      if(Python3_INTERPRETER_ID STREQUAL "Anaconda")
+        set(NOSE_COM "conda' 'activate' 'testing_venv' && ")
+      else()
+        set(NOSE_COM "'source' '${KWIVER_BINARY_DIR}/testing_venv/bin/activate' && ")
+      endif()
+    else()
+      set(NOSE_COM)
+    endif()
+    set(kwiver_test_output_path    "${KWIVER_BINARY_DIR}/$<CONFIG>/bin" )
+  else ()
+    if(VENV_CREATED)
+      if(Python3_INTERPRETER_ID STREQUAL "Anaconda")
+        set(NOSE_COM "conda' 'activate' 'testing_venv' && ")
+      else()
+        set(NOSE_COM "'source' '${KWIVER_BINARY_DIR}/testing_venv/bin/activate' && ")
+      endif()
+    else()
+      set(NOSE_COM)
+    endif()
+    set(kwiver_test_output_path    "${KWIVER_BINARY_DIR}/tests/bin" )
```

Comment:
I could be wrong about this, but this looks like the only difference between the `if(win32)` branch and this `else`?

Reply:
You are correct, and more than likely the `if(VENV_CREATED)` logic can be moved outside the `if(win32)` branch, I've left it there for now because a Windows build w/ an updated CMake version hasn't completed yet, and the logic inside the venv_created section may not actually being identical.